README.md

```diff
@@ -412,6 +413,35 @@ task :foo do
 end
 ```
 
+### Logger
+
+If you want to convert your log messages to Airbrake errors, you can use our
+integration with Ruby's `Logger` class from stdlib. All you need to do is to
+make sure that the integration is required (if the Airbrake is loaded after
+`Logger`, then it is already integrated):
+
+```ruby
+require 'airbrake/logger/logger_ext'
```

Comment:
To conform the general structure that we have in this project and to be able to add more files related to logger in the same directory.